web/libs/editor/src/tags/object/List.jsx

```diff
@@ -10,9 +10,11 @@ import { parseValue } from "../../utils/data";
 import Base from "./Base";
 
 /**
- * The `List` tag is used to display a list of similar items like articles, search results, etc. Task data referred in `value` parameter should be an array of objects with `id`, `title`, and `body` fields.
- * It's much more lightweight than group of other tags like Text. And you can attach classification to provide additional data about this list.
- * Can be used with `Ranker` tag to rank items or pick relevant items from a list.
+ * The `List` tag is used to display a list of similar items like articles, search results, etc. Task data in the `value` parameter should be an array of objects with `id`, `title`, `body`, and `html` fields.
+ * 
+ * It's much more lightweight to use `List` than to group other tags like Text. Also, you can attach classifications to provide additional data about this list.
+ * 
```

Comment:
```suggestion
 *
 * It's much more lightweight to use `List` than to group other tags like Text. Also, you can attach classifications to provide additional data about this list.
 *
```
I remove trailing spaces here